<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimitInputWindow</class>
 <widget class="QDialog" name="LimitInputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Variable to limit:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="limitVar"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Value to limit to:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="limitVal"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>(Use &quot;oo&quot; for infinity)</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>176</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="addMainEquation">
      <property name="text">
       <string>Add to main equation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="addCurVar">
      <property name="text">
       <string>Add to selected variable</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="useMinus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="usePlus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>limitVar</tabstop>
  <tabstop>limitVal</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>usePlus</tabstop>
  <tabstop>useMinus</tabstop>
  <tabstop>addMainEquation</tabstop>
  <tabstop>addCurVar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LimitInputWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LimitInputWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
